### ISSUE TITLE
The link created by Repository - Create desktop icon does not work

### ISSUE THREAD

**Issue by stenci** [git-for-windows/git#448]
Steps to reproduce the problem:
1. Start Git GUI
2. Click Open Existing Repository
3. Browse to a folder with an existing repository and click Select Folder
4. Click Open -> the Git Gui window opens
5. Click on Repository - Create Desktop Icon
6. Click Save -> a Windows shortcut is created

Problem:
The shortcut does not work. Executing the shortcut from a cmd.exe windows shows this:

    C:\Users\myname\Desktop>"Git App.lnk"
    C:\Users\myname\Desktop>/mingw64/libexec/git-core/git-gui: line 3: exec: wish: not found

**Comment by dscho** [comment]
Thank you for the nice [MCVE](http://stackoverflow.com/help/mcve).

As a work-around, you can right-click the shortcut, select Properties, and change the `Target` from

```
"C:\Program Files\Git\usr\bin\sh.exe" -c "CHERE_INVOKING=1 source /etc/profile;'/mingw64/libexec/git-core/git-gui' &"
```

to

```
"C:\Program Files\Git\cmd\git-gui.exe"
```

**Comment by dscho** [comment]
Unfortunately, the fix did not make it into 2.6.0. But you could test it and 2.6.1 (or 2.7.0) will have it.

**Comment by dscho** [comment]
Thanks for testing :stuck_out_tongue_winking_eye:

**Comment by stenci** [comment]
Would testing require installing the SDK and the whole environment? ... sorry, but that's above my paygrade :(

**Comment by dscho** [comment]
> Would testing require installing the SDK and the whole environment? ... sorry, but that's above my paygrade :(

I put in an insane effort to make it dirt-easy to set up the SDK and test things: you simply download and run the Git for Windows SDK installer. It would have required less than 5 minutes of your time, although the computer would have been at work for something like 30 minutes, maybe.

**Comment by stenci** [comment]
I just tried, just to see how it would go. You are right, it is really easy to install, build and run the test suite. Both the installer of the SDK and the instructions work very well.

But I still have not clue what happens when you build. What does make do? I used to program in C in the 80s, but now I can't figure out what happens. 

Does it create git-gui.exe? I couldn't find a file with that name.
Does it create an installer? I couldn't find any .exe with the timestamp that I could use to do my test.

I will keep doing what I know: report errors describing the steps as accurately as possible and leave the rest to you :)
